homework-submission/todoapp/src/Components/todos.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import TodoElement from './TodoElement';
+import { observer, inject } from 'mobx-react';
+
+@inject('todos')
+@observer
+class TodoList extends React.Component {
+  constructor(props) {
+    super(props)
+    props.todos.getTodos()
+  }
+  render() {
+    const { todos } = this.props.todos;
+    const list = todos.map((item, i) => <TodoElement key={item._id} todo={item} />);
```

Comment:
Excellent use of the `_id`, But in not use of the `i`:
```suggestion
    const list = todos.map(item => <TodoElement key={item._id} todo={item} />);
```

?